sqlmesh/utils/__init__.py

```diff
@@ -160,3 +160,8 @@ def double_escape(s: str) -> str:
 
 def nullsafe_join(join_char: str, *args: t.Optional[str]) -> str:
     return join_char.join(filter(None, args))
+
+
+class classproperty(property):
```

Comment:
what is this for can you document this?

Reply:
It is used for Databrick's `can_access_spark_session` class property. Basically it is a property but at the class level. Will add a docstring.